docker-compose.yml

```diff
@@ -24,6 +26,8 @@ services:
     build:
       context: .
       dockerfile: HousingFinanceInterimApi.Tests/Dockerfile
+      args:
+      - LBHPACKAGESTOKEN=${LBHPACKAGESTOKEN}
```

Comment:
This creates the tunnel from host to docker for the token to pass through. But you still need the token itself - for this you need to add the `context: api-nuget-token-context` to your CircleCI workflows.

Reply:
Are you sure? Repairs uses some of the hackney core libraries, yet doesn't include this context

Reply:
I don't know about the repairs, but as far as I'm aware... our Nuget packages are locked behind an auth token due to GH enforcing that. This token needs to come from somewhere regardless of how it comes into the container. The projects I worked with so far, simply use the CCI context. But I suppose you could retrieve it from SSM directly if you wanted to.

Reply:
Or ... it could be that Repairs has set a project-scoped CCI env var for the token. In which case the nuget context wouldn't be needed.

Reply:
do you have an example where this is used?

Reply:
But it's better to use the CCI context than the above mentioned CCI project env var... as it's easier to maintain the token that way.

Reply:
Here: https://github.com/LBHackney-IT/financial-transactions-api/blob/development/.circleci/config.yml#L238-L239